src/formatDate.js

```diff
@@ -51,6 +51,48 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+
+  const splitedDate = date.split(fromFormat[3]);
+  // console.log(splitedDate);
+
+  const indexYYFormat = fromFormat.indexOf('YY');
+  const indexYYYYFormat = fromFormat.indexOf('YYYY');
+
+  const indexYYtoFormat = toFormat.indexOf('YY');
+  const indexYYYYtoFormat = toFormat.indexOf('YYYY');
```

Comment:
```suggestion
  const indexYYFormat = fromFormat.indexOf('YY');
  const indexYYYYFormat = fromFormat.indexOf('YYYY');

  const indexYYtoFormat = toFormat.indexOf('YY');
  const indexYYYYtoFormat = toFormat.indexOf('YYYY');
```
зачем создавать 4 переменные, если результат будет только в двух?